docs/src/index.md

```diff
@@ -10,38 +11,39 @@ Entropies
     [Nonlinear Dynamics](https://link.springer.com/book/10.1007/978-3-030-91032-7),
     Datseris & Parlitz, Springer 2022.
 
-
 In the literature, the term "entropy" is used (and abused) in multiple contexts.
-The API and documentation of Entropies.jl aim to clarify some aspects and
-provide a simple way to obtain probabilities, entropies, or other complexity measures.
+The API and documentation of Entropies.jl aim to clarify some aspects of its usage, and
+to provide a simple way to obtain probabilities, entropies, or other complexity measures.
 
 ### Probabilities
+
 Entropies and other complexity measures are typically computed based on _probability distributions_.
-These are obtained from [Input data for Entropies.jl](@ref) by a plethora of different ways.
-The central API function that returns a probability distribution (in fact, just a vector of probabilities) is [`probabilities`](@ref), which takes in a subtype of [`ProbabilityEstimator`](@ref) to specify how the probabilities are computed.
+These are obtained from [Input data for Entropies.jl](@ref) in a plethora of different ways.
+The central API function that returns a probability distribution (in fact, just a vector of probabilities) is [`probabilities`](@ref), which takes in a subtype of [`ProbabilitiesEstimator`](@ref) to specify how the probabilities are computed.
 All estimators available in Entropies.jl can be found in the [estimators page](@ref probabilities_estimators).
 
 ### Entropies
+
 Entropy is an established concept in statistics, information theory, and nonlinear dynamics.
 However it is also an umbrella term that may mean several computationally different quantities.
 In Entropies.jl, we provide the generic function [`entropy`](@ref) that tries to both clarify the disparate "entropy concepts", while unifying them under a common interface that highlights the modular nature of the word "entropy".
 
 Most of the time, computing an entropy boils down to two simple steps: first estimating a probability distribution, and then applying one of the so-called "generalized entropy" formulas to the distributions.
-Thus, any of the implemented [probabilities estimators](@ref estimators) can be used to compute generalized entropies.
+Thus, any of the implemented [probability estimators](@ref probabilities_estimators) can be used to compute generalized entropies.
```

Comment:
We shouldn't use "probability estimators" if we call the struct `ProbabilitiesEstimator`. Why use plural in one, and singular in the other? In any case, the estimators compute many probabilities, hence plural. No estimator computes a single probability of a single event. (I change it back now)